examples/feedforward_nn.rs

```diff
@@ -7,11 +7,11 @@ fn main() {
     let model = sequential!(
         Ix1,
         [
-            Linear::new(3, 4),
+            Linear::new(String::from("fc1"), 3, 4),
```

Comment:
I already mentioned this in another comment, but this can be changed to `Linear::new("fc1", 3, 4)` after implementing the changes I suggested